app/views/chairs/show.html.erb

```diff
@@ -1,15 +1,16 @@
 <% content_for :title, "Chair #{@chair.name}" %>
 <div class='detail'>
-  <div class="row mb-5">
-    <div class="col-3">
-      <%= image_tag(@chair.image, class: "picture-rounded") %>
-    </div>
-    <div class="col">
-      <h1>
-        <%= @chair.name %>
-      </h1>
+  <div class='row mb-3'>
+    <div class='col-3'> <%= image_tag(@chair.image, class: "picture-rounded") %> </div>
+    <div class="col mx-3">
+      <h2><%= @chair.name %></h2>
```

Comment:
it would better to use an h1 tag here, since h-tags should only be used for semantic reasons
```suggestion
      <h1><%= @chair.name %></h1>
```

Reply:
Since this is a secondary heading and is also used the same way in other show pages shouldn't we stick to this convention?